src/js/tech/html5.js

```diff
@@ -573,6 +577,28 @@ class Html5 extends Tech {
   }
 
   /**
+   * Proxy iOS `webkitbeginfullscreen` and `webkitendfullscreen` into
+   * `fullscreenchange` event
+   *
+   * @private
+   * @method proxyWebkitFullscreen_
+   */
+  proxyWebkitFullscreen_() {
+    if ('webkitDisplayingFullscreen' in this.el_) {
+      const webkitbeginFn = function() {
+        this.one('webkitendfullscreen', function() {
+          this.trigger('fullscreenchange', { isFullscreen: false });
+        });
+
+        this.trigger('fullscreenchange', { isFullscreen: true });
+      };
+
+      this.on('webkitbeginfullscreen', webkitbeginFn);
+      this.on('dispose', () => this.off('webkitbeginfullscreen', webkitbeginFn));
```

Comment:
I don't think `webkitendfullscreen` will be disposed correctly if we dispose the player while in full screen for some reason